app/models/twitter.rb

```diff
@@ -0,0 +1,53 @@
+class Twitter < ActiveRecord::Base
+  require 'rubygems'
+  require 'oauth'
+  require 'json'
+
+  $baseurl = "https://api.twitter.com"
+  $consumer_key = OAuth::Consumer.new(
+    ENV["TWITTER_CONSUMER_KEY"],
+    ENV["TWITTER_SECRET_KEY"]
+  )
+
+  $access_token = OAuth::Token.new(
+    ENV["TWITTER_ACCESS_TOKEN"],
+    ENV["TWITTER_SECRET_TOKEN"]
+  )
+
+  def self.find_tweets(user, since_id)
+    path = "/1.1/statuses/user_timeline.json"
+  	options = Hash.new
+    options["screen_name"] = "#{user}"
+  	options["result_type"] = "recent"
+  	options["count"] = "1"
+  	if since_id
+      options["since_id"] = since_id
+  	end
+  	query = URI.encode_www_form(options)
+  	address = URI("#{$baseurl}#{path}?#{query}&include_rts=false&exclude_replies=true")
```

Comment:
Line is too long. [86/80]<br>Inconsistent indentation detected.<br>Tab detected.